examples/with-realm-web/pages/index.js

```diff
@@ -83,7 +83,7 @@ const IndexPage = () => {
           font-family: sans-serif;
         }
         .status {
-          text-colour: red;
+          text-color: red;
```

Comment:
```suggestion
          color: red;
```

I don't think `text-color` is a valid CSS property?